LRThreeDigits.cpp

```diff
@@ -267,15 +271,20 @@ void updatePortBits(uint16_t *portBits)
 
 void initialize(
     Frequency frequency,
-    Orientation orientation)
+    Orientation orientation,
+    Pins pins)
 {
     // Store the orientation.
     gOrientation = orientation;
+    // Resolve and save the pin offset.
+    if(pins == Pins::From2to11) gPortBitOffset = 2;
```

Comment:
The code style does not match. `{}` brackets are missing and a space after `if`.